src/com/company/Main.java

```diff
@@ -1,40 +1,59 @@
 package com.company;
 
 import java.io.File;
-import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.sql.*;
 import java.util.StringTokenizer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
-public class Main {
+public class Main extends Jdbc{
```

Comment:
Why did you decide to use inheritence here?

Reply:
i thought if there is a multifunctioning then we can use threads 
but here we execute in a single flow so used inheritance

Reply:
instead of inheritance i have created an object and called the methods.